x-pack/plugins/maps/server/register_integrations.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { CoreSetup } from 'kibana/server';
+import { CustomIntegrationsPluginSetup } from '../../../../src/plugins/custom_integrations/server';
+import { APP_ID } from '../common/constants';
+
+export function registerIntegrations(
+  core: CoreSetup,
+  customIntegrations: CustomIntegrationsPluginSetup
+) {
+  customIntegrations.registerCustomIntegration({
+    id: 'ingest_with_gdal',
+    title: i18n.translate('xpack.maps.registerIntegrations.gdal.integrationTitle', {
+      defaultMessage: 'GDAL',
```

Comment:
The screenshot shows something different here. We'll want to follow the guidelines for integrations. `Upload geospatial data with GDAL` sounded good.

Reply:
sorry, that screenshot is out of date. Folded in @gchaps suggestions.